tests/example_app/example/about/our_story/templates/our_story.html

```diff
@@ -0,0 +1,19 @@
+{% extends "layout.html" %}
+
+{% block body %}
+  <br/>
+  <br/>
+  <a href="/about/our-story">Our Story</a>
```

Comment:
Can you make all of the rename suggestions from a previous comment and also use `url_for` instead of hard coding the URL?